external/camel/README.md

```diff
@@ -0,0 +1,20 @@
+# External Camel Tests
+Third Party container tests help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries. For each application, we choose to run a selection of their functional tests.
```

Comment:
Can I get you to add an extra sentence after this one that says:
Camel tests are functional tests pulled from the [camel-quarkus](https://github.com/apache/camel-quarkus.git) repository.  

*please also hyperlink camel-quarkus using the value from github_url in test.properties

Reply:
Done